gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/controller/GraphController.java

```diff
@@ -70,25 +68,14 @@ public class GraphController {
     private GetNamespacesService getNamespacesService;
     @Autowired
     private GetGaaSGraphConfigsService getStoreTypesService;
-    @Autowired(required = false)
-    private AuthService authService;
+//    @Autowired(required = false)
+//    private AuthService authService;
```

Comment:
Should this be commented?

Reply:
Has been deleted. Auth is no longer handled inside rest